docs/GettingStarted.md

```diff
@@ -1844,12 +1869,13 @@ The stats sent will include:
 | `runtime.ruby.thread_count` | `gauge` | Number of threads.                                       |
 | `runtime.ruby.gc.*`.        | `gauge` | Garbage collection statistics (one per value in GC.stat) |
 
-In addition, all metrics will include the following tags:
+Also, all metrics include the following tags:
 
 | Name         | Description                                             |
 | ------------ | ------------------------------------------------------- |
 | `language`   | Programming language traced. (e.g. `ruby`)              |
-| `service`    | List of services this metric is associated with.        |
+| `runtime-id` | Unique identifier of runtime environment (i.e. process) |
```

Comment:
Looks like a merge conflict as well; I think `runtime-id` was removed in the last release.